frontend/src/modules/_shared/components/Toolbar/toolbarDivider.tsx

```diff
@@ -0,0 +1,3 @@
+export function ToolBarDivider(): React.ReactNode {
+    return <div className="w-full h-[1px] bg-gray-300" />;
+}
```

Comment:
True. It could have been part of the other PR with the frontend changes. The plan is to have a general component for the toolbar in the views of `3D viewer`, `Intersection` and `2D viewer` - the latter, however, does not require a divider (yet) while the others do. Either we take it in with this PR or I include it in the next PR for `3D viewer`.

Reply:
If it's to be used in the future, just leave it as it is.